src/frontend/screens/Game/GameSubMenu/index.tsx

```diff
@@ -153,7 +153,7 @@ export default function GamesSubmenu({
         defaultPath: defaultInstallPath
       })
       if (path) {
-        await window.api.changeInstallPath([appName, path, runner])
+        await window.api.changeInstallPath(appName, runner, path)
```

Comment:
If changing it from an array maybe using an object would be better since we won't need to care about the order of the arguments.

Reply:
I think we have a mix of both styles (regular arguments, and a single object containing everything) in the project right now, if we want to unify everything that's fine, but I'd say that belongs into a different PR